manifests/init.pp

```diff
@@ -122,9 +123,13 @@
       group  => 'root',
     }
     file { '/var/lib/influxdb':
-      ensure => directory,
-      owner  => 'influxdb',
-      group  => 'influxdb',
+      ensure  => directory,
+      owner   => 'influxdb',
+      group   => 'influxdb',
+      require => [
```

Comment:
This is auto-require dependency, not necessary

Reply:
Thanks, I was wondering why this worked without the `require`.